### ISSUE TITLE
confusing related synonyms for adaptors

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#26041]
Is it confusing  to have "protein complex scaffold" terms as related synonyms of "molecular adaaptor"

Researchers I speak to seem to view "scaffolding" and "adapting" as differnet activities appear to view a protein complex scaffold more as a structural molecule activity.

Conversation with a researcher this week:
"distinction between adaptor and scaffold and we have very good examples with our work: TTT is an adaptor in the sense that it recruits and temporarily stabilizes interactions between a chaperone and its substrates. These interactions are by nature weak and transient and happen cotranslationally (at least in S. pombe: PMID: 34686329). In contrast, a scaffolding subunit is a subunit that resides in the complex and is essential to hold it together – without it the other subunits disassemble and are often destabilized. This is the case in SAGA for Spt7 and, Ada1 (Hfi1) for example."

The particular example  above is in the context of a co-chaperone (TTT) which acts as a chaperone substrate adaptor (but could also apply to other substrate adaptors).  This distinction would not solve our problem completely because in the context of a membrane adaptor  the transient nature would not apply. So some adaptors are not transient (although I think they would be described as transient when compared to complexes, which appear to usually be very stable after assemlby). My problem is that there are plenty of protein complex subunits which are described as scaffolds, but that it would be difficult to describe as adaptors, so these activities are not interchangable.

Any suggestions on how to annotate protein complex scaffolding subunits? Should I use structural molecule activity? If so should we have a more specific term, and should  protein complex scaffold activity be added to this term?

It would be very useful  good to discuss on an annotation call.
I can add plenty of examples of complexes where I have this annotation problem.

**Comment by ValWood** [comment]
At the same time we should discuss, in the context of chaperone adaptors which are not part of the final complex, if they should be annotated to the  processes that the complex is involved in (personally I would not, these are indirect phenotypes because the mature protein does not fold correctly, and is then not assembled into the complex at all, or incorrectly). These poteis have no role in regulatin the process in a 'normal' situation.

Here is an example:
https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=Q9NWS0
(I will list the indirct terms when I get chance, and challenge the original annotations and the PAINT transfer)

This is unsurprising since the  GO  Complex term 

GO:0097255   R2TP complex
Definition 
A highly conserved protein complex comprised of two ATP-dependent DNA helicases (Rvb1p and Rvb2p in yeast, Pontin52 and Reptin52 in humans), Pih1p in yeast or PIH1D1 in humans, and Tah1 in yeast or RPAP3 in humans. The complex associates with Hsp90 and is thought to have a role in assembly of large protein or protein/nucleic acid complexes. In this role it is involved in multiple processes such as box C/D snoRNP biogenesis, phosphatidylinositol-3 kinase-related protein kinase (PIKK) signaling, RNA polymerase II assembly, and others. PMID:15766533 PMID:21925213

Implies that it is OK to annotate to these downstream processes:
phosphatidylinositol-3 kinase-related protein kinase (PIKK) signaling
Even though TTT is a co-translational chaperone.

Nice summary here:
https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01334-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124721013346%3Fshowall%3Dtrue

https://www.cell.com/cms/attachment/26c358d2-4bdb-4cde-8725-4a4eaf33908e/gr7.jpg
and in figure 7 (which shows the various complexes that TTT chaperones)


Also, need to consider that (Rvb1p and Rvb2p in yeast, Pontin52 and Reptin52 in humans)
 have an additional role in some complexes because they are part of the final assemlby. However this isn't the case for  the "phosphatidylinositol-3 kinase-related protein kinase", so any annotations to this (or TOR signalling) are indirect)

**Comment by ValWood** [comment]
MMs19 is another example. Scaffod, adaptor , or something else:

see fig 7 [https://www.sciencedirect.com/science/article/pii/S0021925820417314](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS0021925820417314&data=05%7C01%7Cvw253%40universityofcambridgecloud.onmicrosoft.com%7Cbda837915fee4963df7c08dbb5f6ba98%7C49a50445bdfa4b79ade3547b4f3986e9%7C1%7C0%7C638303843519066413%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=806u32eHzbAiul48WoVrSOHpzebjX5yEl6NqFLgOc%2FQ%3D&reserved=0)

**Comment by ValWood** [comment]
Can we discuss this on a future annotation call? @vanaukenk @pgaudet

**Comment by Antonialock** [comment]
....I would think something like an E3 ligase is an adaptor (?), whereas scaffolds are either proteins whose sole purpose is to glue stuff together in a complex, or to organize a cascade of events (eg see below)

![image](https://github.com/geneontology/go-ontology/assets/12727506/6fc855c6-ae3b-46fd-862e-9e54ffa882d9)

**Comment by vanaukenk** [comment]
@ValWood  - happy to put this on an agenda for a future annotation call.  If tomorrow is too soon, we could plan for next week?  Let me know.

**Comment by ValWood** [comment]
Tommorrow might be too soon.
I had a discussion with a user today  who does a lot of biochemistry around chromatin remodeling complexes and SAGA.
He had some useful suggestions. I will plug them in here tomorrow and we'll see if there is a conflict wuth what we have.

I think my major issue is if an adaptor has 2 inputs, sometimes and is an adaptor to another adaptor the models are going to get a bit crazy. I have better examples, just need to dig them out.

It's important to ge this right becasue so many  gene products function as adaptors

**Comment by vanaukenk** [comment]
Sounds fine @ValWood 
We could at least point curators to this ticket for them to start thinking about this issue?  We would likely have a better chance of getting good feedback if people have time to think/prepare ahead.

**Comment by ValWood** [comment]
I agree with the above comments, but this isn't necessarily how the terms are represented in the ontology. I tried to summarize here and provide some questions and examples:

https://docs.google.com/document/d/1Qakmsk19Sr4H_2cDpzMU0bi8duD7Rbye5aJ_Rne-7ZE/edit?usp=sharing

I started a doc. I will flesh it out over the next couple of weeks but I'm really busy for the next 2 weeks with meetings and reviewing etc.
If you have any  more questions or answers please add with 'track changes'

The problems aren't huge here, the ontology and the annotation best practice just requires a little of clarification to make sure we are consistent.

**Comment by ValWood** [comment]
I think the ontology part is easy (clear distinction between scaffold and adaptor) , but I'm also  interested how we should model substrates for GO-CAM when an adaptor has 2 inputs. Or if we only need to model the substrate/client. I think that is the gist of it.

**Comment by ValWood** [comment]
Out of date. A couple of bits I will put in a fresh ticket.

**Comment by ValWood** [comment]
Note that there is now a "protein complex scaffold activity " term
added here:
https://github.com/geneontology/go-ontology/issues/31444
